_sass/footer.scss

```diff
@@ -1,2 +1,45 @@
 /* Footer Styles */
 
+footer {
+	max-height: 265px;
```

Comment:
This max-height can be deleted. The content in .footer-nav creates the necessary height on its own in the DOM without the need for us to specifically calculate the height manually.

Reply:
Good catch. I have removed this.